src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -74,6 +76,10 @@ def do_work():
         # Device Global Manager
         DeviceGlobalCfgMgr(common_objs, "CONFIG_DB", swsscommon.CFG_BGP_DEVICE_GLOBAL_TABLE_NAME),
     ]
+
+    if device_info.get_platform_info().get('switch_type') == "voq":
```

Comment:
we need not limit to voq chassis but to any chat chasssi.

Please replace the API to this one: https://microsoft-my.sharepoint.com/:w:/p/abdosi/EYYnnVcex0dHhhpNGvb5jXUBSxZ2XHanHxOmwAJf0ZxiFQ?e=uKdM57

Reply:
Thank you for your comment and for pointing out the correct API. Done